xblockutils/base_test.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import time
+
+import pkg_resources
+
+from django.template import Context, Template
+
+from selenium.webdriver.support.ui import WebDriverWait
+
+from workbench import scenarios
+from workbench.test.selenium_test import SeleniumTest
+
+from .resources import ResourceLoader
+
+
+class SeleniumBaseTest(SeleniumTest):
```

Comment:
Just as an FYI in #5 I have developed a replacement for `SeleniumBaseTest`, called `SeleniumXBlockTest`, which is largely similar but faster and more flexible. It also doesn't have the `time.sleep()` hack. I'd encourage using it instead of this class for any new code.